client_app/src/components/volunteer/VolunteerDashboard.js

```diff
@@ -23,20 +30,21 @@ function VolunteerDashboard() {
         <h3>Impact Created</h3>
         <div className="card">
           <h4>Total hours served</h4>
-          <p>106</p>
+          <p>{impactData.totalHours}</p>
         </div>
         <div className="card">
           <h4>Lives Touched</h4>
-          <p>17</p>
+          <p>Too many to count</p>
         </div>
         <div className="card">
           <h4>Shelters served</h4>
-          <p>4</p>
+          <p>{impactData.sheltersServed}</p>
```

Comment:
Maybe all of this can be moved to Impact.js, where you call the backend API to retrieve the past shifts and calculate the hours and shelters.